### ISSUE TITLE
[REVIEW] SuperAayush

### ISSUE THREAD

**Issue by SuperAayush** [EddieHubCommunity/support#3817]
### GitHub Profile Link

https://github.com/SuperAayush

**Comment by schmelto** [comment]
Hey @SuperAayush it would be good if you had a customised GH profile you can checkout Eddie's free course for that.

**Comment by SuperAayush** [comment]
Thank you so much for the suggestion @schmelto, after watching yesterday's live stream and some awesome profiles like yours I am also working on building some awesome readme for my profile as well and surely will follow Eddie's free course.

**Comment by schmelto** [comment]
> Thank you so much for the suggestion @schmelto, after watching yesterday's live stream and some awesome profiles like yours I am also working on building some awesome readme for my profile as well and surely will follow Eddie's free course.
> 
>  

Nice! If you are ready feel free to ping me here or elsewhere I would like to provide some feedback :)

**Comment by SuperAayush** [comment]
Hey, @schmelto can you have a look at my profile now and provide me with some nice feedback 😄.

**Comment by schmelto** [comment]
## Profile Review
### TLDR;

Your profile looks awesome! See the full details with my suggestions below.

---

### Full details

Your profile looks awesome! Here some suggestions from my side:

* remove the {} from the headings I think there are not needed
* Like the Idea with giving green squares away maybe there is a way to automate this

**Comment by SuperAayush** [comment]
What does TLDR stands for 😅?
Ya, I am also working on automating the issues part or maybe any other way to give out green squares.

**Comment by schmelto** [comment]
> What does TLDR stands for 😅?
> 
> Ya, I am also working on automating the issues part or maybe any other way to give out green squares.

To long didn't reed

**Comment by eddiejaoude** [comment]
Looks great, some thoughts...
- animated gif seems designed for dark mode
- snake green squares does not work so well and makes your profile longer
- pinned repos to have descriptions
- try to do more code reviews
